acme_diags/derivations/acme.py

```diff
@@ -604,7 +604,8 @@ def cosp_histogram_standardize(cld):
         (('QFLX',), lambda qflx: qflxconvert_units(qflx))
     ]),
     'LHFLX': OrderedDict([
-        (('hfls',), rename)
+        (('hfls',), rename),
+        (('QFLX'), lambda qflx: qflxconvert_units(qflx))
```

Comment:
Is this the appropriate derivation?

Reply:
Yes! By searching through the derived variable list: `NET_FLUX_SRF`, will also impacted by lacking of `LHFLX`, Can you please and an entry for `NET_FLUX_SRF` with `QFLX`.

Reply:
Is that correct? For instance, `FLDSC` on https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/derivations/acme.py#L495 can be derived from `FLNSC`, which itself can be derived from `sfc_net_lw_clr_mon`, yet we do not list this for `FLDSC` also. 

Are you saying `NET_FLUX_SRF` should have a line where the `LHFLX` in https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/derivations/acme.py#L521 is replaced with `QFLX`? That seems redundant.

Reply:
Not replaced, but in addition, to add a line using QFLX instead of LHFLX to cover the case only LHFLX is present